ig/input/fsh/nbs-obr-segment.fsh

```diff
@@ -14,8 +14,7 @@ Description: "The OBR Segment"
 * universalServiceIdentifierNameOfCodingSystem 1..1 SU string "OBR-4.3 Universal Service Identifier Name Of Coding System"
 * universalServiceIdentifierOriginalText 1..1 SU string "OBR-4.9 Universal Service Identifier Original Text"
 * observationDateTime 1..1 SU dateTime "OBR-7 Observation Date/Time"
-// other optional fields
-
+* specimenActionCode 0..1 SU string "OBR-11 Specimen Action Code"
```

Comment:
Perhaps just add in the description here that the field is currently hardcoded to `P`

Reply:
@sfradkin I was wondering where/how to put that. Can I just append it to `OBR-11 Specimen Action Code`? Something like `OBR-11 Specimen Action Code (value is always 'P')`?

Reply:
I think that's fine.  We might need to look for a place to add a longer description (I think there's a way to do this in FSH) to indicate _why_ it's hardcoded to `P`